source/extensions/filters/http/jwt_authn/jwks_async_fetcher.cc

```diff
@@ -86,6 +101,7 @@ void JwksAsyncFetcher::onJwksSuccess(google::jwt_verify::JwksPtr&& jwks) {
 void JwksAsyncFetcher::onJwksError(Failure) {
   ENVOY_LOG(warn, "{}: failed", debug_name_);
   handleFetchDone();
+  refetch_timer_->enableTimer(RefetchAfterFailedSec);
```

Comment:
checking if this will do a retry back off and where the retry_policy is implemented for RemoteJwks

Reply:
OnJwksError will be called from after retry failed in [jwks_fetcher](https://github.com/envoyproxy/envoy/blob/9f80d2c2c4a561dfb8b65a7fb27589ea1bc0abeb/source/extensions/filters/http/common/jwks_fetcher.cc#L68)

So this retry is on top of this [retry_policy](https://github.com/envoyproxy/envoy/blob/287ccb9daafb3df3f42f6d83bbffd965c49cddb9/api/envoy/extensions/filters/http/jwt_authn/v3/config.proto#L366)